modules/tinymce/src/themes/silver/main/ts/ui/button/MenuButton.ts

```diff
@@ -42,7 +42,7 @@ const renderMenuButton = (spec: MenuButtonSpec, prefix: string, backstage: UiFac
   tooltip: spec.tooltip,
   // https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-2/menubar-2.html
   role,
-  fetch: (callback) => {
+  fetch: (comp, callback) => {
```

Comment:
Was wondering if `comp` should have an underscore to be clear it isn't used?

Reply:
Fixed